.eslintrc.js

```diff
@@ -1,32 +1,21 @@
 module.exports = {
-  extends: [
-    'airbnb',
-    'plugin:prettier/recommended',
-    'prettier/react',
-    'plugin:@typescript-eslint/recommended'
-  ],
-  plugins: ['react', 'prettier', 'import'],
-  parser: '@typescript-eslint/parser',
+  extends: ['@verypossible/eslint-config/native'],
+  plugins: ['import'],
+  env: {
+    node: true,
```

Comment:
Had to do this because the linter yelled with a `'module' is not defined.` message on configuration files (like this one). Thoughts?

Reply:
I pushed a change to the config that includes this `env` across the board -- I don't see why not. Makes life easier too.

You should be able to reinstall it -- v1.0.2

Reply:
All good, since this is the first time we're using the config it does make sense to make those changes, then we can focus on a more solid process 🕺